lib/filter/enemy.js

```diff
@@ -99,7 +99,31 @@ exports.update = function (json) {
       }
     }
     */
-
+   
+		console.log( 'Enemy Drops:');
+		var drops = 0;
+		for (enemy in json[round].enemy) {
+			if (json[round].enemy[enemy].children[part].drop_item_list.length > 0) {
+				drops = 1;
+				for (var item in json[round].enemy[enemy].children[part].drop_item_list) {
```

Comment:
The reason for the 2 loops is: Normal enemies appears in multiple "enemy" parts, but bosses with multiple forms/enemies appear as multiple "children" of a single "enemy". That's why both are array.